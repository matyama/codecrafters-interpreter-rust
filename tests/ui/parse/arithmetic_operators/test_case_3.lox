42 * 65 * 78 / 69
