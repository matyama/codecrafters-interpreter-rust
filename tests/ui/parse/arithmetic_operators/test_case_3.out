(/ (* (* 42.0 65.0) 78.0) 69.0)
